C/Coin-OR/CSDP/build_tarballs.jl

```diff
@@ -3,6 +3,7 @@ include("../coin-or-common.jl")
 name = "CSDP"
 version = v"6.2.0"
 
+
```

Comment:
In this file you could replace the `all_load` dance with the new shiny [`flagon`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Utilities-in-the-build-environment-1) script! :grin: